src/pages/test-report/components/controlled-stop/controlled-stop.scss

```diff
@@ -0,0 +1,72 @@
+controlled-stop {
+
```

Comment:
is this all just copied over from competency? If so, can we used `@extend` instead?

Reply:
90% of it is, theres a few changes in sizes. I think we need to refactor the css for all the buttons . Legal Requirements & ETA's also do the same , so we have pretty much the same css 4 times.

Are you happy for me to do this as a second PR?

Reply:
of course